src/Mailgun/Api/Pagination.php

```diff
@@ -10,6 +10,7 @@
 namespace Mailgun\Api;
 
 use Mailgun\Assert;
+use Mailgun\Resource\Api\mixed;
```

Comment:
Whoops, didn't mean to do that! Did a find-and-replace on PaginationResponse to get rid of the type hints and set the annotations to mixed, but I guess I missed one :) nice catch!